src/views/notifications/BlockRewardNotification/index.tsx

```diff
@@ -6,6 +6,7 @@ import { useStringGetter } from '@/hooks/useStringGetter';
 
 import { Details } from '@/components/Details';
 import { Icon, IconName } from '@/components/Icon';
+// eslint-disable-next-line import/no-cycle
```

Comment:
:'(

Reply:
I will fix these soon, I struggle to untangle what exactly is happening in here